packages/source-drupal/src/DrupalSource.js

```diff
@@ -0,0 +1,201 @@
+const axios = require('axios')
+const reduce = require('lodash.reduce')
+const { 
+  cullByWordCount
+} = require('./utils')
+const slugify = require('@sindresorhus/slugify')
+const Nodes = require('./entities/Nodes')
+const TaxonomyTerms = require('./entities/TaxonomyTerms')
+const Files = require('./entities/Files')
+const Users = require('./entities/Users')
+
+
+class DrupalSource {
+  // defaultOptions merged with this.options in App.vue
+  static defaultOptions() {
+    return {
+      baseUrl: '',
+      views: [], // deprecated
+      entities: [],
```

Comment:
```suggestion
      entities: ['node', 'taxonomy_term', 'file', 'user'],
```

I think we should allways fetch these types as default. And maybe the option should only add more types to the array.

Reply:
@hjvedvik the `defaultsDeep` in App.js that merges the `defaultOptions` with the user options doesn't do a concat of the Array. It replaces the `indexes`.

Does this need to be adjust in App.js first?

Reply:
I see.. The option should probably be empty if we add them to the default types. You can merge the arrays inside `processEntities` instead.

Reply:
Right. Good call.

Reply:
https://github.com/gridsome/gridsome/pull/41/commits/470568165e6178aaa4bfbbfd62bfdd6804e6476b